client/agenda/AgendaScheduleList.vue

```diff
@@ -296,7 +296,7 @@ const meetingEvents = computed(() => {
           color: 'red'
         })
       }
-      if (agendaStore.useNotes) {
+      if (agendaStore.uses_notes) {
```

Comment:
I think this one needs `usesNotes` also?

Reply:
dang.  but no change in test result, so I missed it.  fix pushed.